apps/fluent-tester/src/TestComponents/Common/iconExamples.tsx

```diff
@@ -1,13 +1,30 @@
 import { Platform } from 'react-native';
-import { SvgIconProps, FontIconProps } from '@fluentui-react-native/icon';
-import TestSvg from '../Button/test.svg';
+import { SvgIconProps, FontIconProps, IconProps } from '@fluentui-react-native/icon';
+import TestSvg from '../../../assets/test.svg';
+import { useTheme } from '@fluentui-react-native/theme-types';
 
 export const svgProps: SvgIconProps = {
   src: TestSvg,
   viewBox: '0 0 500 500',
 };
 
-const testTtf = require('./../../../../assets/Font_Awesome_900.otf');
+export const commonIconProps: IconProps = { svgSource: svgProps };
+// In some cases, android icons need a color explicitly passed to render. This hook gets an updated version of the props above with a color option.
+export function useCommonIconProps(color?: string | { light: string; dark: string }): IconProps {
+  const {
+    host: { appearance },
+  } = useTheme();
+  const ret: IconProps = { svgSource: svgProps };
```

Comment:
Can we wrap this instance in a useMemo? Right now it creates a new object on every hook run (every render), which will cause unnecessary rerenders of the icon.

Reply:
You could probably include the color logic in the useMemo too.

Reply:
Removed the hook for android-specific styles as per above.